src/components/dashboards/Admin-dashboard.tsx

```diff
@@ -1,20 +1,100 @@
 "use client";
 import { useSession } from "next-auth/react";
 import { useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
+
+interface TimeSlot {
+  id: string;
+  startTime: string;
+  endTime: string;
+  description: string;
+}
+
+interface Ticket {
+  id: string;
+  status: string;
+  requestedTimeSlot: TimeSlot;
+  user: User;
+}
```

Comment:
Same with this one, from `@prisma/client`.
Also we are calling this object `ResidentRequest`